coins/ethereum/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "ethereum"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+thiserror = "1"
+serde_json = "1.0"
+serde = "1.0"
+hex-literal = "0.3"
+ethers = { git = "https://github.com/gakonst/ethers-rs", features = ["abigen", "ethers-solc"] }
+eyre = "0.6"
```

Comment:
This appears to be a fork of anyhow? It'd be eligible for usage in processor, intended to be the app, yet this should just use thiserror.

Reply:
Oh. That's... stupid. In that case, sure, keep it if you prefer it. I didn't want to introduce it, but if its ethers, its ethers. We may have to do the thiserror *somewhere* though, and this may be the best place to do so? Depends on how things play out... Regardless, keep for now, resolve later?